SFR/inner_fuel_pincell/make_SFR_pincell_model.py

```diff
@@ -0,0 +1,136 @@
+import openmc
+import numpy as np
+from argparse import ArgumentParser
+import openmc.material
+import warnings
+from materials import *
+
+warnings.filterwarnings("ignore")
+
+
+def make_sfr_material(material_dict, density: float, percent_type: str):
+    material = openmc.Material()
+    for nuclide, material_info in material_dict.items():
+        material.add_nuclide(
+            nuclide, percent=material_info["percent"], percent_type=percent_type
+        )
+
+    return material
+
+
+def settings(shannon_entropy: bool, height: float):
+    pitch = 1.25984
```

Comment:
I think it would be worth considering setting a default for shannon_entropy and checking/raising an exception to make sure height is provided and positive.

pitch is a magic number here